### ISSUE TITLE
Clicking on "Refresh Status" in CRC setup page results in indefinite loading 

### ISSUE THREAD

**Issue by lgrossma** [redhat-developer/vscode-openshift-tools#4413]
https://github.com/user-attachments/assets/de7915cb-077c-48fe-acce-e396ed0e928a

**Comment by vrubezhny** [comment]
For me it kook about 30 minutes to install and try starting the CRC cluster, which actually didn't succeeded:
```
$ crc status -o json
{
  "success": false,
  "error": "Cannot get machine state: unexpected libvirt status 3\n",
  "preset": ""
}

But the Refresh Status never hang up for me. 
```

**Comment by lgrossma** [comment]
Yeah, the startup takes a long time even when starting the crc in terminal, but via terminal the crc start always ends with success for me while it fails sometimes in the extension.

**Comment by vrubezhny** [comment]
@lgrossma When you see a startup failure, please attach the log from Openshift Local logs Output  here. I expect some error to occur so the startup cannot succeed. I'm interested if your log contains any error statuses like:
```
{
  "success": false,
  "error": "...",
  ...
}
```

On indefinite loading after "Refresh Status" is clicked... I cannot reproduce the issue, nor get the effect you screened on you video (dropping the settings and sending it back to the CRC install/initialization wizard when "Back" is clicked during the refreshing... 
The refresh actually invokes `crc version` + `crc status` then `crc console --credentials`, so when one of the first  two commands `hangs up` it could result into an infinite loading... But I couldn't reproduce this on a normally working CRC cluster. 
When I was trying it previously on a "problematic" CRC installation (it couldn't be started probably due to lack of available disk space) - refreshing was taking much longer, but still was finishing successfully (showing the stopped cluster)
